app/src/main/java/mubstimor/android/codelab/CircleTransform.java

```diff
@@ -0,0 +1,45 @@
+package mubstimor.android.codelab;
+
+import android.graphics.Bitmap;
+import android.graphics.BitmapShader;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+
+import com.squareup.picasso.Transformation;
+
+/**
```

Comment:
Please work me through what you were trying to achieve here

Reply:
I was trying to add a transformation to change the image from a default square into a circular one. This script was written as an extra to the picasso library.

Reply:
Okay, @mubstimor Please remove that comment and add something more descriptive.

Reply:
@joeeasy I have updated the comment.